command/schedule_cmd.go

```diff
@@ -642,6 +643,67 @@ func DeleteScheduleOverrideAction(c *gcli.Context) {
 	renderResponse(c, resp, err)
 }
 
+func GetOnCallsAction(c *gcli.Context){
+	cli := NewScheduleClient(c)
+	req := &schedule.GetOnCallsRequest{}
+	if scheduleName, ok := getVal("name",c); ok {
+		req.ScheduleIdentifierType = schedule.Name
+		req.ScheduleIdentifier = scheduleName
+	} else if scheduleID, ok := getVal("id",c);ok {
+		req.ScheduleIdentifierType = schedule.Id
+		req.ScheduleIdentifier = scheduleID
+	}
+	if atTime, ok := getVal("atTime", c); ok {
+		t, err := time.Parse(time.RFC3339, atTime)
+		exitOnErr(err)
+		req.Date = &t
+	}
+	flat := c.IsSet("flat")
+	req.Flat = &flat
+
+	resp, err := cli.GetOnCalls(context.Background(), req)
```

Comment:
Curious, what does `context.Background` do? I don't see it being used by other `cmd`s.

Reply:
It would be better to send a default context than nil